homeassistant/components/nextbus/manifest.json

```diff
@@ -6,5 +6,5 @@
   "documentation": "https://www.home-assistant.io/integrations/nextbus",
   "iot_class": "cloud_polling",
   "loggers": ["py_nextbus"],
-  "requirements": ["py-nextbusnext==1.0.0"]
+  "requirements": ["py-nextbusnext==1.0.1"]
```

Comment:
This exception argument still says sequence:
https://github.com/ViViDboarder/py_nextbusnext/blob/8d73f9af867a4779a165ee85532affce2cea72a6/py_nextbus/client.py#L253

Is the order of stops not important for the multi stop api call?

Reply:
I just double checked. No. Order is not important.

Reply:
I have no idea how I managed to use a set here and then explicitly write a test in the library for disallowing sets… This should resolve it anyway.

Reply:
Oh, I'll fix that error message too so it's not confusing.